aospy/calc.py

```diff
@@ -767,3 +767,19 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+    def addAttrs(self, data):
+        if (isinstance(data, xr.DataArray)):
+            self.addAttrsDataArray(data)
+        else:
+            for name, da in data.data_vars.items():
+                self.addAttrsDataArray(da)
+
+    def addAttrsDataArray(self, data):
+        if self.var.units != '':
+            if self.var.dtype_out_vert == 'vert_int':
+                data.attrs['units'] = '(vertical integral of {0}): {0} kg m^-2)'.format(self.var.units)
```

Comment:
I think this line is longer than [79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length); one way to break it up would be to define the string first in a separate variable, then format and assign to `data.attrs['units']` in a separate line:
```python
units = '(Vertical integral of {0}): {0} kg m^-2'
data.attrs['units'] = units.format(self.var.units)
```

Reply:
Similar to my comment below re: always adding a description even if empty, I think we should do the same thing for units.

One question, though, is what units to use for vertical integrals when the original units are empty.  As written, we don't include them.  My initial thought is to, in that case, do e.g. `(Vertical integral of quantity with unspecified units)`